go/libraries/doltcore/doltdb/foreign_key_coll.go

```diff
@@ -75,6 +74,27 @@ func (f *ForeignKeyViolationError) Error() string {
 	}
 }
 
+// getRowValuesAsString returns row values as string format.
+// Variables key and val have tags in them, so tags are removed, and blobs are converted appropriately
+func getRowValuesAsString(key types.Value, val types.Value) string {
+	valSlice, _ := val.(types.Tuple).AsSlice()
+	all, _ := key.(types.Tuple).Append(valSlice...)
+	allSlice, _ := all.AsSlice()
+	var rowVals []string
+	for i, s := range allSlice {
+		if i%2 == 1 {
+			switch v := s.(type) {
+			case types.Blob:
+				rowVals = append(rowVals, v.DebugText())
```

Comment:
I'm not sure this is the best way to handle this. What does MySQL return when setting up this same scenario and using a MySQL client? `DebugText` seems like the wrong thing to use here.

Reply:
MySQL client returns 
```
ERROR 1452 (23000): Cannot add or update a child row: a foreign key constraint fails (`tempdb`.`#sql-577f_a`, CONSTRAINT `fk_b_a_id_refs_a` FOREIGN KEY (`a_id`) REFERENCES `a` (`id`))
```